packages/features/portfolio/src/containers/portfolio-state-zero/hooks/use-card-data.tsx

```diff
@@ -9,73 +8,27 @@ import {
   PointsIcon,
   VaultIcon,
 } from '@notional-finance/icons';
-import {
-  useAllMarkets,
-  useAllVaults,
-  useSelectedNetwork,
-} from '@notional-finance/notionable-hooks';
+import { useSelectedNetwork } from '@notional-finance/notionable-hooks';
 import { PORTFOLIO_STATE_ZERO_OPTIONS, PRODUCTS } from '@notional-finance/util';
 import { FormattedMessage } from 'react-intl';
-import { useTokenData } from './use-token-data';
+import { sumAndFormatIncentives } from '@notional-finance/shared-web/dashboard-view/hooks/utils';
+import { getAvailableVaults } from './use-network-token-data';
 
 export const useCardData = (
-  selectedTab: number,
+  selectedTabIndex: number,
   activeToken: string,
-  allTokenData: ReturnType<typeof useTokenData>['allTokenData'],
-  availableSymbols: string[]
+  tokenData: any,
```

Comment:
Using `any` here is not good, you should be able to get the return type using typescript utility functions. If not, use the AI helper to generate the proper types.